src/types.d.ts

```diff
@@ -0,0 +1 @@
+declare module "burner-provider";
```

Comment:
whats up with this? looks like its from the UerProvider hook -- I dont think we use either (they are artifacts from scaffold-eth afaik) -- can you confirm if this and UserProvider are needed and if not feel free to delete

Reply:
oh you are right, this is not used, will delete these two files :)